### PR TITLE
return match result

### PR DESCRIPTION
**Pull request by akutil**

### CODE REVIEW

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 1]
```diff
@@ -15,6 +15,10 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        int total = (bet1 > bet2) && (result1 > result2) ? 1 : 0;
+        total = (bet1 < bet2) && (result1 < result2) ? 1 : total;
+        total = (bet1 == bet2) && (result1 == result2) ? 1 : total;
+        total = (result1 + " " + result2).equals(bet1 + " " + bet2) ? 2 : total;
+        return total;
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FootballMatch.java]: it's too complicated.Try to solve it without creating redundant variables and with less number of '?' conditions.

Comment (Andrii-Kutil) [reply src/main/java/core/basesyntax/FootballMatch.java]: done

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 2]
```diff
@@ -15,6 +15,10 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        int winTeamNumber = result1 > result2 ? 1 : result1 < result2 ? 2 : 0;
+        int myPrediction = bet1 > bet2 ? 1 : bet1 < bet2 ? 2 : 0;
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FootballMatch.java]: do not create redundant variables

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 3]
```diff
@@ -15,6 +15,10 @@
...
+        return (result1 + "" + result2).equals(bet1 + "" + bet2) ? 2 : winTeamNumber == myPrediction ? 1 : 0;
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FootballMatch.java]: it's too complicated, try to simplify this solution

Comment (dmytro-hryhoruk) [reply src/main/java/core/basesyntax/FootballMatch.java]: do not push before mvn clean package shows "Success"

Comment (akutil) [reply src/main/java/core/basesyntax/FootballMatch.java]: Done. I understood my problem. I had to group same conditions together.

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request closed without merging**